### PR TITLE
WIP: Add icon mapping

### PR DESCRIPTION
**Pull request by micstein89**
Extension of the code to map emission inventories (starting with the TNO inventory) onto ICON grids.

### CODE REVIEW

**Comment by mjaehn** [comment]
Could you implement the following suggestion: https://github.com/C2SM-RCM/cosmo-emission-processing/issues/6#issuecomment-666264979

**Comment by micstein89** [comment]
> 
> 
> Could you implement the following suggestions? [#6 (comment)](https://github.com/C2SM-RCM/cosmo-emission-processing/issues/6#issuecomment-666264979) and [#1 (comment)](https://github.com/C2SM-RCM/cosmo-emission-processing/issues/1#issuecomment-667071515)

The central_longitude as fct of pollon and the additional line in the README is implemented with the latest commit.

**Comment by mjaehn** [comment]
What is the status here? Is further work required or is everything working as intended? For the latter case, the "WIP" can be removed.

**Comment by micstein89** [comment]
This pull request can be closed.

File: emiproc/__init__.py [hunk 1]
```diff
@@ -231,10 +231,15 @@ def process_tno(cfg, interpolation, country_mask, out, latname, lonname):
...
-                                                       source_type=t)
+                            				source_type=t)
                         print('Write as variable:', out_var_name)
-                        util.write_variable(out, out_var, out_var_name,
-                                            latname, lonname, unit)
+                        if cfg.model.startswith("cosmo"):
+                            util.write_variable(out, out_var, out_var_name,
+                                                latname, lonname, unit)
+                        elif cfg.model.startswith("icon"):
+                            print(out_var.shape)
```
Comment (gkuhl) [inline emiproc/__init__.py:240]: You might want to delete this `print(out_var.shape)`, which was probably added for debugging?

File: emiproc/grids.py [hunk 2]
```diff
@@ -761,3 +761,222 @@ def intersected_cells(self, corners):
...
+#                continue
+#            icon_cell_xmin = min(self.vlon[self.vertex_of_cell[:,n]-1])
+#            if icon_cell_xmin > lon:
+#                continue
+#            icon_cell_ymax = max(self.vlat[self.vertex_of_cell[:,n]-1])
+#            if icon_cell_ymax < lat:
+#                continue
+#            icon_cell_ymin = min(self.vlat[self.vertex_of_cell[:,n]-1])
+#            if icon_cell_ymin > lat:
+#                continue
```
Comment (gkuhl) [inline emiproc/grids.py:872]: Probably not necessary to keep these commented lines?

**Review by gkuhl — COMMENTED** [review]
Looks fine for merging. Just two small comments where you might want to remove a print statement and remove some lines that were commented out.

File: emiproc/grids.py (hunk 2 above)
Comment (micstein89) [reply emiproc/grids.py]: Done

Comment (gkuhl) [reply emiproc/grids.py]: Thanks

**Pull request merged**